test/connector/http/basicauth/docker-compose.yml

```diff
@@ -31,6 +31,8 @@ services:
     command: wget -q -O- nginx:8080/
     depends_on:
       - secretless
+    environment:
+      - COMPOSE_INTERACTIVE_NO_CLI:1
```

Comment:
can we add a comment to explain why this is needed here? the same way we did in evoke

Reply:
oops, didn't mean to leave this in, just had it there for testing